ManageWikiNamespaces.php

```diff
@@ -387,4 +387,22 @@
 		'help' => '',
 		'requires' => [],
 	],
+	'wgMFNamespacesWithoutCollapsibleSections' => [
+		'name' => 'Disable collapsible sections in this namespace?',
+		'from' => 'mobilefrontend',
+		'type' => 'check',
+		'main' => true,
+		'talk' => true,
+		'excluded' => [],
+		// See https://github.com/wikimedia/mediawiki-extensions-MobileFrontend?tab=readme-ov-file#wgmfnamespaceswithoutcollapsiblesections
+		'overridedefault' => [
```

Comment:
I'm unsure if (a) this is the right format or (b) it's good practice to override the default like this rather than just allow the `mobilefrontend` extension to set the default. Advice very much appreciated!